adback-solution-to-adblock/includes/class-ad-back.php

```diff
@@ -165,7 +165,7 @@ private function defineAdminHooks()
         $this->loader->addAction( 'wp_ajax_dismiss_adback_incentive', $plugin_admin, 'dismissAdbackIncentive');
 
         // Add admin notice if isn't dismissed
-        if (empty(get_option('adback-incentive-dismissed'))) {
+        if (get_option('adback-incentive-dismissed') === false) {
```

Comment:
It seem to me that the second parameter to get_option is the default value : https://developer.wordpress.org/reference/functions/get_option/